azure-pipelines.yml

```diff
@@ -70,7 +70,7 @@ stages:
           echo "Either job kvmtest-t0-part1 or job kvmtest-t0-part2 failed! Please check the detailed information."
           exit 1
         fi
-
+        
```

Comment:
This is not new line, this is new line with spaces, you should remove the spaces.